tests/test_config.py

```diff
@@ -169,6 +169,20 @@ def test_getboolean_error(self):
         with self.assertRaises(ValueError):
             self.cli_config.getboolean(section, option)
 
+    def test_items_case_insensitive(self):
+        section = 'MySection'
+        self.cli_config.set_value(section, 'option', 'value')
+        self.cli_config.set_to_use_local_config(True)
+        self.cli_config.set_value(section, 'Option', 'localValue')
+        items_result = self.cli_config.items(section)
+        self.assertEqual(len(items_result), 1)
+        self.assertEqual(items_result[0]['value'], 'localValue')
+
+        with mock.patch.dict('os.environ', {self.cli_config.env_var_name(section, 'OPTION'): 'envValue'}):
```

Comment:
Better to use raw string, instead of `env_var_name` to test the functionality more thoroughly. Consider:

```py
self.assertEqual(env_var_name(), env_var_name())
```